src/object.d

```diff
@@ -1526,41 +1526,73 @@ const:
 
     @property uint flags() nothrow pure @nogc { return _flags; }
 
+    /************************
+     * Returns:
+     *  module constructor for thread locals, `null` if there isn't one
```

Comment:
I feel many of these requests do not improve the quality. This might be a case that warrants a quick merge. I am all for it.